### PR TITLE
[SKIP CI] fixup: restore ability to build with a non-default manifest

### PR DESCRIPTION
**Pull request by marc-hb**
5 commits, look only at the last one. All the others belong to #6005 

See (too) long discussion in https://github.com/thesofproject/sof/pull/6005

### CODE REVIEW

File: scripts/xtensa-build-zephyr.py [hunk 1]
```diff
@@ -615,13 +581,21 @@ def main():
...
 
-	# we made two modes mutually exclusive with argparse
-	if args.west_path:
-		# check west_path input + create symlink if needed
-		run_point_mode()
-	if args.clone_mode:
+	west_root_dir, west_manifest_path = find_west_workspace()
+	if not west_root_dir:
+		execute_command(["west", "init", "-l", f"{SOF_TOP}"], cwd=west_top)
+		west_root_dir, west_manifest_path = find_west_workspace()
```
Comment (aborisovich) [inline scripts/xtensa-build-zephyr.py:587]: We do not need to find west workspace here, we had just created one one line above.

Comment (marc-hb) [reply scripts/xtensa-build-zephyr.py:587]: This is needed to correct the `print()`s below.

**Comment by lgirdwood** [comment]
@marc-hb are you meaning the fixup patch to be squashed ?

**Comment by marc-hb** [comment]
@lgirdwood this was just a demo for the long #6005 discussion. It's not relevant anymore, closing.

PS: please don't waste your precious and limited time on drafts :-)

**Pull request closed without merging**